lib/ansible/cli/galaxy.py

```diff
@@ -98,6 +146,13 @@ def init_parser(self):
                                 help='The path to the directory containing your roles. The default is the first '
                                      'writable one configured via DEFAULT_ROLES_PATH: %s ' % default_roles_path)
 
+        default_collections_path = C.config.get_configuration_definition('COLLECTIONS_PATHS').get('default', '')
+        collections_path = opt_help.argparse.ArgumentParser(add_help=False)
+        collections_path.add_argument('-p', '--collections-path', dest='collections_path', type=opt_help.unfrack_path(pathsep=True),
```

Comment:
Do you want to open a separate PR that does this since this is more common code we both need?

Reply:
Sure thing!

Reply:
#66969